modules/nw-http2-haproxy.adoc

```diff
@@ -17,40 +17,4 @@ The connection from HAProxy to the application pod can use HTTP/2 only for re-en
 [IMPORTANT]
```

Comment:
From the bug [comment](https://issues.redhat.com/browse/OCPBUGS-18820?focusedId=23006662&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23006662) I think the statement in Line 15 should be updated.
```
The implication is that end-to-end HTTP/2 is possible with passthrough and re-encrypt and not with insecure or edge-terminated routes
```
We should remove `insecure or` from the statement, add note that "HTTP/2 using an insecure route is possible whether the ingress controller has HTTP/2 disabled or enabled".